pattern-lab/.devtools-watch-ignore

```diff
@@ -0,0 +1,4 @@
+.*___jb_tmp___
+.*___jb_old___
+\.*sw*
+
```

Comment:
Well, I didn't know this file was a thing. Seems like I should add this to p2-env to exclude `\build`